CHANGELOG.adoc

```diff
@@ -106,7 +106,7 @@ It is the responsibility of the drop targets (e.g. a diagram) to validate the dr
 - https://github.com/eclipse-sirius/sirius-web/issues/2718[#2718] Replacing useNodes and useEdges by useReactFlow to imporove performances.
 - https://github.com/eclipse-sirius/sirius-web/issues/2787[#2787] [view] Add the ability to customize the icons of the node tools and edge tools in the View DSL.
 - https://github.com/eclipse-sirius/sirius-web/issues/2784[#2784] [performance] Expand monitoring capabilities.
-
+- https://github.com/eclipse-sirius/sirius-web/issues/2790[#2790] [diagram] Increase the size of the handles used to resize a node.
```

Comment:
We should probably add an entry in breaking changes to warn users who have created their own custom node that they can apply the same correction

Reply:
Optional, but maybe also a screenshot showing before/after?

Reply:
Thanks for the review, I added a screenshot and a line in breaking changes